src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -498,6 +500,10 @@ private RowMapBuffer getTransactionRows(BinlogConnectorEvent beginEvent) throws
 					String sql = qe.getSql();
 					String upperCaseSql = sql.toUpperCase();
 
+					if (sql.startsWith(BinlogConnectorEvent.XAEND)) {
+						LOGGER.info("XA END");
+						return buffer;
+					}
```

Comment:
please move this into `.isCommitEvent()`, and then let the logic on line 458 trigger.

Reply:
OK!I think is a greater idea!

Reply:
Now! I think _XA END_ event is not belong to _isCommitEvent()_.
only _XA COMMIT_ event and _XA ROLLBACK_ belong to.